phones/tests/mgmt_delete_phone_data_tests.py

```diff
@@ -0,0 +1,272 @@
+from datetime import datetime, timezone, timedelta
+from io import StringIO
+from typing import Iterator
+from unittest.mock import Mock, patch
+
+from django.contrib.auth.models import User
+from django.core.management import call_command, CommandError
+from django.conf import settings
+
+from allauth.socialaccount.models import SocialAccount
+from model_bakery import baker
+from pytest_django.fixtures import SettingsWrapper
+import pytest
+
+if settings.PHONES_ENABLED:
+    from ..models import InboundContact, RealPhone, RelayNumber
+
+pytestmark = pytest.mark.skipif(
+    not settings.PHONES_ENABLED, reason="PHONES_ENABLED is False"
+)
+THE_COMMAND = "delete_phone_data"
+
+
+@pytest.fixture(autouse=True)
+def test_settings(settings: SettingsWrapper) -> SettingsWrapper:
+    """Override settings for tests"""
+    settings.SUBSCRIPTIONS_WITH_PHONE = ["PHONE_SUBSCRIPTION"]
+    return settings
+
+
+@pytest.fixture(autouse=True)
+def mock_twilio_client() -> Iterator[Mock]:
+    """Mock PhonesConfig with a mock twilio client"""
+    with patch("phones.apps.PhonesConfig.twilio_client") as mock_twilio_client:
+        yield mock_twilio_client
+
+
+@pytest.fixture
+def phone_user(db: None, test_settings: SettingsWrapper) -> User:
```

Comment:
suggestion (blocking): let's reconcile this with [the other utility functions in `model_tests.py`](https://github.com/mozilla/fx-private-relay/blob/eae992837a2c0a007591ec2803d2548919b635ec/phones/tests/models_tests.py#L77-L99).